src/server/pfs/server/server_v2.go

```diff
@@ -0,0 +1,40 @@
+package server
+
+import (
+	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
+	"github.com/pachyderm/pachyderm/src/server/pkg/serviceenv"
+)
+
+func NewObjClient(conf *serviceenv.Configuration) (objClient obj.Client, err error) {
```

Comment:
We could probably clean this up a bit by just returning the client creation, rather than using named return values. Also, lets set dir using `dir := config.StorageRoot`. That is a more concise initialization and is more consistent with the codebase.